atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -260,6 +261,34 @@ trait AtumImplicitsBase {
       dataset
     }
 
+    /**
+     * The method returns ControlMeasure object from the Atum context
+     * @return - ControlMeasure object
+     */
+    def getControlMeasure: ControlMeasure = {
+      atum.preventNotInitialized()
+      atum.controlFrameworkState.accumulator.getControlMeasure
+    }
+
+    /**
+     * The method returns AdditionalInfo object from the Atum context
+     * @return - AdditionalInfo object
+     */
+    def getAllAdditionalInfo: Map[String, String] = {
+      atum.preventNotInitialized()
```

Comment:
Good pattern is to use the function already created to get the value you are after, instead of redoing the code.
E.g. use `getControlMeasure` here.

Reply:
Yes, I completely agree with this point. I should have realized this 🦆 .